garnbarn_api/models.py

```diff
@@ -68,9 +68,10 @@ class Assignment(models.Model):
     """Data and detail about assignment"""
 
     # The assignment shouldn't get deleted when tag is deleted.
-    tag = models.ForeignKey(Tag, on_delete=models.SET_NULL, null=True)
+    tag = models.ForeignKey(Tag, on_delete=models.SET_NULL,
+                            null=True, blank=True, default=None)
     author = models.ForeignKey(
-        CustomUser, on_delete=models.CASCADE, null=True, default=None)
+        CustomUser, on_delete=models.CASCADE, blank=True, null=True, default=None)
```

Comment:
Should author allow to be None?

Reply:
I think It shouldn't. but actually, you are the one who adds this change in commit fa223f5 . So I'll give this question back to you.

Reply:
For now I think we can leave it like this.